alpha/alpha-server/pom.xml

```diff
@@ -75,6 +75,10 @@
       <groupId>org.apache.servicecomb.pack</groupId>
       <artifactId>alpha-core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
This change could introduce some trouble if we want to replace the service registry.

Reply:
If we support consul in the future, we can add the consul client dependency and then use which parameter control e.g

eureka.enabled=false
consul.enabled=true